.github/workflows/Tests.yaml

```diff
@@ -868,6 +888,8 @@ ${{ env.CACHE_KEY_SUFFIX }}"
       # documentation so we don't need to do that here.
       - name: Commit and push
         run: |
+          # Workaround git https://github.com/actions/checkout/issues/760
+          git config --global --add safe.directory $GITHUB_WORKSPACE
```

Comment:
This is possibly unnecessary because it's not in the container, but probably doesn't hurt either. This step is hard to test.

Reply:
Yea, I decided to go with the "I'd rather not need it and have our builds work again then spend more time trying to get it perfect" if that's alright with you :)